docker-compose.yml

```diff
@@ -23,6 +23,10 @@ x-deployment-env: &deployment-env
   SG_ENV: ${SG_ENV:-development}
   TZ: $TZ
 
+x-edge-env: &edge-env
+  PLACE_EDGE_SECRET: ${PLACE_EDGE_SECRET:-ZWRnZTEyMzRfczNjcjN0}
```

Comment:
What's the source of that secret?

Reply:
I don't like this part of the flow, and edge will never be deployed like this.

An `Edge` is created in backoffice and its token should be placed in the environment of `edge`.

It wouldn't be too hard to create a task that spits out a token similar to how the secret for influxdb is generated.

Reply:
For immediate needs, could this just be a blatantly obvious placeholder / dev-only secret? A neater approach to generation can then be built as part of #8.

Reply:
I've removed the placeholder, for that reason I'll leave in the `restart: "no"` to prevent it pegging CPU with restarts